### PR TITLE
Add Trusted Node Policy

### PR DESCRIPTION
**Pull request by lumjjb**
Signed-off-by: Brandon Lum <lumjjb@gmail.com>

### CODE REVIEW

File: community/CM-Configuration-Management/policy-trusted-node.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+apiVersion: policy.open-cluster-management.io/v1
+kind: Policy
+metadata:
+  name: policy-trusted-node
+  annotations:
+    policy.open-cluster-management.io/standards: NIST SP 800-53
+    policy.open-cluster-management.io/categories: CM Configuration Management
+    policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+spec:
+  complianceType: musthave
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-trusted-node.yaml]: this field is not needed

File: community/CM-Configuration-Management/policy-trusted-node.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  remediationAction: inform
+  disabled: false
+  namespaces:
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-trusted-node.yaml]: this field is not needed

File: community/CM-Configuration-Management/policy-trusted-node.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    include: ["default"]
+  policy-templates:
+    - complianceType: musthave
```
Comment (ycao56) [inline community/CM-Configuration-Management/policy-trusted-node.yaml]: this field is not needed

**Review by ycao56 — APPROVED** [review]

**Comment by lumjjb** [comment]
Changes made, btw these fields are still in policy-trusted-container

**Pull request merged**